tests/baselines/reference/capturedLetConstInLoop5.errors.txt

```diff
@@ -235,7 +238,8 @@ tests/cases/compiler/capturedLetConstInLoop5.ts(229,13): error TS2365: Operator
             (() => x + y + v);
             if (x == 1) {
                 ~~~~~~
-!!! error TS2365: Operator '==' cannot be applied to types '0' and '1'.
+!!! error TS2367: The types of these values indicate that this condition will always be 'false'.

```

Comment:
`These values are known to be '0' and '1', rendering this condition always 'false'.`
 ^^^^ how much smoother to comprehend and fix!